doc/source/advanced.rst

```diff
@@ -180,14 +178,13 @@ For example:
 
 .. ipython:: python
 
-   # original MultiIndex
-   df.columns
+   df.columns  # original MultiIndex
 
-   # sliced
-   df[['foo','qux']].columns
+   df[['foo','qux']].columns  # sliced
 
 This is done to avoid a recomputation of the levels in order to make slicing
-highly performant. If you want to see the actual used levels.
+highly performant. If you want to see only the used levels, you can use the
+`get_level_values()` method.
```

Comment:
if you reference a method, use ``:func:`MultiIndex.get_level_values` ``

Reply:
Done